src/victory-util/transitions.js

```diff
@@ -249,15 +311,19 @@ export function getTransitionPropsFactory(props, state, setState) {
       const exitingNodes = childTransitions && childTransitions.exiting;
       const exit = transitionDurations.exit || getChildTransitionDuration(child, "onExit");
       // if nodesWillExit, but this child has no exiting nodes, set a delay instead of a duration
-      const animation = exitingNodes ? {duration: exit} : {delay: exit};
-      return onExit(exitingNodes, data, assign({}, animate, animation));
+      // devide duration to apply half of the duration time to clipPath animation
+      // , half to shape animation
+      const animation = exitingNodes ? {duration: exit / 2} : {delay: exit / 2};
```

Comment:
I'm not certain this is correct. When the other exit animations are just the identity, I think the clipPath animation should take the entire duration